### ISSUE TITLE
Exception on clicking "GPS Feature Import" in Spatial Unit Manager

### ISSUE THREAD

**Issue by nyalldawson** [gltn/stdm#395]
Clicking the "GPS Feature Import" button in the spatial unit manager leads to an AttributeError exception: 

```
AttributeError: 'NoneType' object has no attribute 'current_user' 
Traceback (most recent call last):
  File "C:/Users/Nyall/.qgis2/python/plugins\stdm\ui\spatial_unit_manager.py", line 1020, in on_import_gpx_file_button_clicked
    self.active_sp_col
  File "C:/Users/Nyall/.qgis2/python/plugins\stdm\ui\gps_tool.py", line 50, in __init__
    self._init_entity_editor()
  File "C:/Users/Nyall/.qgis2/python/plugins\stdm\ui\gps_tool.py", line 112, in _init_entity_editor
    self.entity_editor = EntityEditorDialog(self.entity, self.model, self)
  File "C:/Users/Nyall/.qgis2/python/plugins\stdm\ui\forms\editor_dialog.py", line 172, in __init__
    self.current_user = parent.entity_browser.current_user
AttributeError: 'NoneType' object has no attribute 'current_user'
```

This is unrelated to the recent QGIS 3 porting work -- I can reproduce it on fe7c37fe2f848007589bf9d4d374418eaa4e9ff1

**Comment by pgathogo** [comment]
This issue is resolved but I've not pushed it upstream.

**Comment by nyalldawson** [comment]
@pgathogo have you got a link to a commit which fixes it? I'm happy to pull it into the qgis 3 work...

**Comment by pgathogo** [comment]
@nyalldawson, please see the fix on this branch [34bb848](https://github.com/gltn/stdm/commit/34bb8481e437db41e35aa37747bbed1df511fd96)
